### ISSUE TITLE
Can't define save path [Windows]

### ISSUE THREAD

**Issue by Dogway** [Parchive/par2cmdline#180]
I'm unable to make this work in almost all conditions in Windows.

Don't Work:
Error log: "You must specify a list of files when creating."
`"D:\exepath\par2.exe" c -s5120000 -r5 -v "Q:\AnotherPath\PAR2\filename.par2" "Q:\FilePath\filename.mp4"`
`"D:\exepath\par2.exe" c -s5120000 -r5 -v -B "Q:\FilePath" "Q:\AnotherPath\PAR2\filename.par2" "filename.mp4"`

Do Work:
`"D:\exepath\par2.exe" c -s5120000 -r5 -v "filename.par2" "filename.mp4"`
`"D:\exepath\par2.exe" c -s5120000 -r5 -v "Q:\FilePath\filename.mp4"`

In other words, I'm unable to choose where to save the par2 files.

Might be related to https://github.com/Parchive/par2cmdline/issues/169
